content/02.introduction.md

```diff
@@ -5,41 +5,42 @@
 
-[Git](https://git-scm.com/) is the version control system that enables all the collaborative tools available on GitHub.
+Git is the version control system that enables all the collaborative tools available on GitHub.
 In Git, versions of files and directories are uniquely identified as "commits", allowing one to identify and track modifications line-by-line.
-Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see (Box 1)[#box-1-definitions]) is necessary, the GitHub web-based platform and its integrated development environments (such as the GitHub Desktop) allow users to manage their repositories without using more technical command-line sessions. 
-We do not focus on Git in this paper, but we recommend users explore the many resources providing detailed information on Git, such as journal articles [@doi:10.1371/journal.pcbi.1004668; @doi:10.1371/journal.pcbi.1004947], video tutorials, and books [@https://happygitwithr.com/].
+Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see Box @box:box-1-definitions) is necessary, the GitHub web-based platform and its integrated development environments (such as the GitHub Desktop) allow users to manage their repositories without using more technical command-line sessions. 
+We do not focus on Git in this paper, but we recommend users explore the many resources providing detailed information on Git, such as journal articles [@doi:10.1371/journal.pcbi.1004668; @doi:10.1371/journal.pcbi.1004947], video tutorials, and books [@https://happygitwithr.com].
 
 <!-- ### General background on how GitHub is used by EEB researchers -->
 <!-- *Contributors to this section: RCO, PHPB* -->
 The voluminous user-community and the numerous resources providing pedagogical instruction material on how to use GitHub streamlined its widespread use (CITE).
 Nevertheless, although researchers in EEB have been encouraged to adopt GitHub as part of their research process [@doi:10.1038/s41559-017-0160; @doi:10.1038/538127a], its use is still not widespread.
 Because GitHub and its features have been centered on collaboration for software development in information systems [@doi:10.1109/ASONAM.2016.7752419], first-time users from domains without formal training in information technology may face steep learning curves.
-Moreover, domain-specific perspectives and resources providing tractable examples and practical guidance for researchers in ecology and evolution on GitHub are scarce (but see https://ourcodingclub.github.io/, https://www.openscapes.org/, ADD OTHER CITATIONS).
+Moreover, domain-specific perspectives and resources providing tractable examples and practical guidance for researchers in ecology and evolution on GitHub are scarce (but see [@https://ourcodingclub.github.io; @https://www.openscapes.org]).
```

Comment:
@BrandonEdwards Thanks for catching all these https citation issues!

Reply:
I think this is maybe going to render with double citations:
>... are scarce (but see (author year; author year)). 

removing the square brackets should cause it to render like:
> ... are scarce (but see author (year); author (year))

Not sure how citations for URLs get rendered though so I guess lets wait and see what Manubot does.